ISSUE-TRIAGE.md

```diff
@@ -0,0 +1,106 @@
+exist and it is not easy to parse the error message.
+
+**Proposed solution**:
+
+docker rm should have consistent exit codes for different types of errors so
+that the user can easily script and know the reason why the command failed.
+
+```
+
+### Step 3: Classify the Issue
+
+Classifications help both to inform readers about an issue's priority and how to resolve it.
+This is also helpful for identifying new, critical issues.  "Kinds of" are
+applied to the issue or pull request using labels.  You can apply one or more labels.
+
+
+Kinds of classifications:
+
+| Kind             | Description                                                                                                                     |
+|------------------|---------------------------------------------------------------------------------------------------------------------------------|
+| kind/enhancement | Enhancement are not bugs or new features but can drastically improve usability or performance of a project component.           |
+| kind/cleanup     | Refactoring code or otherwise clarifying documentation.                                                                         |
+| kind/content     | Content that is not documentation such as help or error messages.                                                               |
+| kind/graphics    | Work involving graphics skill                                                                                                   |
+| kind/regression  | Regressions are usually easy fixes as hopefully the action worked previously and git history can be used to propose a solution. |
+| kind/bug         | Bugs are bugs. The cause may or may not be known at triage time so debugging should be taken account into the time estimate.    |
+| kind/feature     | Functionality or other elements that the project does not currently support.  Features are new and shinny.                      |
+| kind/question    | Contains a user or contributor question requiring a response.                                                                   |
+| kind/usecase     | A description of a user or contributor situation requiring a response perhaps in code or documentation.                         |
+| kind/writing     | Writing documentation, man pages, articles, blogs, or other significant word-driven task.                                       |
+| kind/test        | Tests or test infrastructure needs adding or updating                                                                           |
+| priority/PN      | Describes the priority of the issue                                                                                             |
+| area/AreaName    | Areas like test, mesos, doc, discovery, network etc                                                                             |
+
+Contributors can add labels by using a `+kind/bug` in an issue or pull request comment.
```

Comment:
Does this still work? Does this work in this repository? I know it worked at some point during the Docker Birthday Party, but not sure that's still true, perhaps someone should test